sigs.yml

```
@@ -0,0 +1,508 @@
+    tcSponsors:
+    - name: Carlos Alberto
+      github: carlosalberto
+    - name: Jack Berg
+      github: jack-berg
+    gcLiaison: []
+  - name: 'Specification: Events'
+    meeting: Every Friday at 10:00 AM PT
+    notes:
+      type: gDoc
+      value: 1BKjQWP32FXL9g1cGbyj7DMXV1Uq_RL8_78rWaMBhN0A
+    chat:
+      - type: slack
+        name: '#otel-event'
+        id: C062HUREGUV
+    invites: calendar-spec-events
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Semantic Conventions: General'
+    meeting: Every Monday at 08:00 PT
+    notes:
+      type: gDoc
+      value: 10xG7DNKWRhxNmFGt3yYd3980a9uwS8lMl2LvQL3VNK8
+    chat:
+      - type: slack
+        name: '#otel-semantic-conventions-wg'
+        id: C041APFBYQP
+    invites: calendar-semconv
+    tcSponsors:
+    - name: Armin Ruech
+      github: arminru
+    - name: Josh Suereth
+      github: jsuereth
+    - name: Reiley Yang
+      github: reyang
+    gcLiaison:
+    - name: Trask Stalnaker
+      github: trask
+  - name: 'Semantic Conventions: Messaging'
+    meeting: Every Thursday at 8:00 PT
+    notes:
+      type: gDoc
+      value: 1dWHhyXnfVife-cQ2DW5-d5Ldp1Lq8Rre2UsHpyo8cEE
+    chat:
+      - type: slack
+        name: '#otel-messaging'
+        id: C02Q4AAHDSA
+    invites: calendar-semconv
+    tcSponsors: []
+    gcLiaison:
+    - name: Trask Stalnaker
+      github: trask
+  - name: 'Semantic Conventions: System Metrics'
+    meeting: Every Thursday at 08:30 PT
+    notes:
+      type: gDoc
+      value: 1p5TH57t43XpxA48onLzX4PIr3g6ydYKCtR_AUlsCnQk
+    chat:
+      - type: slack
+        name: '#otel-system-metrics-wg'
+        id: C05CTFE9U4A
+    invites: calendar-semconv
+    tcSponsors:
+    - name: Josh Suereth
+      github: jsuereth
+    gcLiaison: []
+  - name: 'Semantic Conventions: LLM'
+    meeting: Every Wednesday alternating between 08:00 and 16:00 PT
+    notes:
+      type: gDoc
+      value: 1EKIeDgBGXQPGehUigIRLwAUpRGa7-1kXB736EaYuJ2M
+    chat:
+      - type: slack
+        name: '#otel-llm-semconv-wg'
+        id: C06KR7ARS3X
+    invites: calendar-semconv
+    tcSponsors:
+    - name: Liudmila Molkova
+      github: lmolkova
+    gcLiaison: 
+    - name: Alolita Sharma
+      github: alolita
+- name: Implementation SIGs
+  sigs:
+  - name: Maintainer Meeting
+    meeting: Every Monday at 09:00PT
+    notes:
+      type: gDoc
+      value: 1_X2CTiDXmxk2JozBNwQ2uP8FfV1rxx9ISE5puHu0yIk
+    chat:
+      - type: slack
+        name: '#otel-maintainers'
+        id: C01NJ7V1KRC
+    invites: calendar-maintainer-meeting
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Android: SDK + Automatic Instrumentation'
+    meeting: Every Tuesday at 08:00PT
+    notes:
+      type: gDoc
+      value: 1W72oElAJuYx3efM9wkH5l1iksdfwmTSkVl-H7KHTLnY
+    chat:
+      - type: slack
+        name: '#otel-android'
+        id: C05J0T9K27Q
+    invites: calendar-android
+    tcSponsors: []
+    gcLiaison:
+    - name: Trask Stalnaker
+      github: trask
```

I realized this SIG meeting conflicts with the Specification meeting which would make it hard for me to attend.

```suggestion
    gcLiaison: []
```